pcap.pyx

```diff
@@ -403,7 +408,7 @@ def lookupdev():
     p = pcap_ex_lookupdev(ebuf)
     if p == NULL:
         raise OSError, ebuf
-    return p
+    return p.decode('UTF-8')
```

Comment:
Same as above we should be sure to mention that we are changing the API and returning `unicode` instead of `str`.